clojurians_log/playbooks/roles/clojure/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Do we need this role? As we are installing Leiningen already and would probably be building an uberjar from the source and running it with `java -jar ...`. Leiningen will get the Clojure jar automatically when building the project. What do you think @plexus ?

Reply:
The clojure role installs the clojure CLI tools (the clj and clojure commands).

Reply:
I think we can remove it and use it when we actually need it. What do you think?

Reply:
OK. The clojure role is removed.